app/helpers/container_build_helper/textual_summary.rb

```diff
@@ -0,0 +1,78 @@
+module ContainerBuildHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name creation_timestamp resource_version)
+  end
+
+  def textual_group_relationships
+    %i(ems container_project )
+  end
+
+  def textual_group_smart_management
+    items = %w(tags)
+    items.collect { |m| send("textual_#{m}") }.flatten.compact
+  end
+
+  def textual_group_build_instances
+    h = {:labels => [_("Name"), _("Phase"), _("Message"), _("Reason"),
+                     _("Pod"), _("Output Image"),
+                     _("Start Timestamp"), _("Completion Timestamp"), _("Duration"),
+                    ]
+        }
+    h[:values] = @record.container_build_pods.collect do |build_pod|
+      [
+        build_pod.name,
+        build_pod.phase,
+        build_pod.message,
+        build_pod.reason,
+        link_to_pod(build_pod.container_group),
+        build_pod.output_docker_image_reference,
+        build_pod.start_timestamp,
+        build_pod.completion_timestamp,
+        parse_duration(build_pod.duration / 1_000_000_000),
+      ]
+    end
+    h
+  end
+
```

Comment:
IMHO, it would be valuable to present also the labels (in a different table), like we do for replicators, services, pods and nodes:

```
def textual_container_labels
  textual_key_value(@record.labels.to_a)
end
```
```
def textual_key_value(items)
  items.collect { |item| {:label => item.name.to_s, :value => item.value.to_s} }.flatten.compact
end
```